core/nodejsActionBase/package.json

```diff
@@ -2,10 +2,10 @@
   "name": "nodejsAction",
   "version": "0.0.1",
   "dependencies": {
-    "body-parser": "1.15.2",
+    "body-parser": "1.18.3",
     "btoa": "1.1.2",
-    "express": "4.14.0",
+    "express": "4.16.3",
     "log4js": "0.6.38",
-    "request": "2.79.0"
+    "request": "2.87.0"
```

Comment:
This new package versions for these packages will not take affect for user actions.
They also need to be updated in the Dockerfile here https://github.com/apache/incubator-openwhisk-runtime-nodejs/blob/master/core/nodejs6Action/Dockerfile#L41